### DISCUSSION TITLE
Chat stops updating when streaming big response

### DISCUSSION THREAD

**Discussion by mootfrost** [danny-avila/LibreChat#3983] [Troubleshooting]
### What happened?

When response is being generated, chat suddenly stops updating IRL and throws error in devtools console:
`Uncaught (in promise) SyntaxError: JSON.parse: unexpected non-whitespace character after JSON data at line 1 column 1788 of the JSON data`.
The only related logs i found is warning in nginx.

### Steps to Reproduce

1. Deploy LibreChat using deploy-compose file
2. Open new chat
3. Send prompt that requires long response(ex/ Explain Knuth-Morris-Pratt algorithm)
4. Wait to response to be generated

### What browsers are you seeing the problem on?

Chrome

### Relevant log output

```shell
[warn] 34#34: *28349 an upstream response is buffered to a temporary file /var/cache/nginx/proxy_temp/2/63/0000000632 while reading upstream, client: <redacted>, server: localhost, request: "GET /assets/vendor-CJ1DHdVT.js HTTP/1.0", upstream: "http://172.28.0.6:3080/assets/vendor-CJ1DHdVT.js", host: "<redacted>"
```

### Screenshots

![image](https://github.com/user-attachments/assets/50baaa4c-5183-401d-8ea8-8d205cfcea8c)

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by danny-avila** [comment]
Not reproducible, and looks to be an NGINX configuration issue so I moved this to Discussions. Could be related to the network transfer size limit that is configurable for NGINX: https://github.com/danny-avila/LibreChat/discussions/3061#discussioncomment-9776776

It makes sense because a longer response can be a larger size with the legacy stream handlers.

**Reply by danny-avila** [reply]
Also related: https://github.com/danny-avila/LibreChat/discussions/3960

Example of long conversation using `deploy-compose.yml` as container setup, with no issues:
https://demo.librechat.ai/share/MaJZDUVGjXyQX6Y6c4DeR

**Comment by mootfrost** [comment]
@danny-avila  Still happens every time when `index.4Mdi5Eba.js` is not still cached(in incognito mode) even with nginx bufferization disabled

**Reply by danny-avila** [reply]
If the response takes a long time to generate, it could also be a timeout issue. NGINX will close a connection that is over 60 seconds running, and this can be extended.
